### DISCUSSION TITLE
Please alter the contributing guidelines to be more inclusive

### DISCUSSION THREAD

**Discussion by ethindp** [zeroc-ice/ice#1427] [General]
Hi all,

I am interested in contributing to Ice and I understand that signing the contributor agreement is a requirement. I have noticed that electronic signatures are not accepted and a handwritten signature is required. Handwritten signatures are a traditional method, but electronic signatures are legally binding as well. I would like to inquire about the reason for this requirement and its intent.

I have a visual impairment, which makes it difficult for me to comply with the handwritten signature requirement. I understand that this requirement may not have been designed with individuals with disabilities in mind, and I kindly request that it be altered or removed in order to be more inclusive. The contributor agreement is in PDF format, which can be accessed using screen readers with some limitations.

I hope that a solution can be found that will allow individuals with disabilities to contribute to Ice as well. I am not seeking special treatment, but rather a permanent solution that will ensure that everyone has the opportunity to contribute to the project without any barriers. Thank you for your understanding.

**Comment by ReeceHumphreys** [comment]
Thank you for reaching out and for your interest in contributing to Ice. We appreciate your feedback and concern about the requirement for a handwritten signature on the contributor agreement.

We're actively looking into alternative solutions to make contributions to the Ice project easier for everyone, no matter what challenges they face. We'll keep you updated in this discussion on any changes we make. Our goal is to create a welcoming and accessible environment for all contributors. Thank you again for your input and understanding!

**Comment by ReeceHumphreys** [comment]
Hello @ethindp!

I have an update about the ZeroC contributor agreement (ZCA). We have migrated to using a CLA bot that will allow you to review and sign the ZCA entirely through the browser. Let me know if you have any issues or additional questions!

**Comment by ethindp** [comment]
@ReeceHumphreys Thank you for taking care of this issue so promptly. I appreciate it!
